ui/tests/acceptance/job-detail-test.js

```diff
@@ -292,7 +300,15 @@ moduleForJob(
 moduleForJob(
   'Acceptance | job detail (service)',
   'allocations',
-  () => server.create('job', { type: 'service', noActiveDeployment: true }),
+  () =>
+    server.create('job', {
+      type: 'service',
+      noActiveDeployment: true,
+      withPreviousStableVersion: true,
+      allocStatusDistribution: {
+        running: 1,
+      },
```

Comment:
You might be wondering about allocStatusDistribution! In this case, there's a test that checks the (old-style, historical) job summary view and tries to click the visualization for the first kind of allocations it sees. It looks like this:

<img width="803" alt="image" src="https://github.com/user-attachments/assets/839e3ec9-2520-4449-89a8-e402fd6b71f7" />
^-- So in this one, it would click the "2 complete" link and continue the test.

WELL!!! If the only allocations that the mock job creator has created are of `Queued` type, there is no link! Because why would there be a link! Because they are queued. We are creating a simulacrum and it has forgotten the face of its father.

Reply:
this comment was an adventure :grinning: